RecoMuon/L3TrackFinder/interface/HLTMuonL2SelectorForL3IO.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RecoMuon_HLTMuonL2SelectorForL3IO_HLTMuonL2SelectorForL3IO_H
+#define RecoMuon_HLTMuonL2SelectorForL3IO_HLTMuonL2SelectorForL3IO_H
+
+/**  \class HLTMuonL2SelectorForL3IO
+ * 
+ *   L2 muon selector for L3 IO:
+ *   finds L2 muons not previous converted into L3 muons
+ *
+ *   \author  Benjamin Radburn-Smith - Purdue University
+ */
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+
+namespace edm {class ParameterSet; class Event; class EventSetup;}
```

Comment:
Hi Slava, What does this actually mean? If the class is in the same package is it ok, but if it is from a different package it should not be forward declared? I see that 3.7 states that if from the same package you can forward declare if sufficient.